bot/tests/test_lando.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you rename this file `test_reporter_lando.py`? This way we match the names of the other tests for the other reporters.